content/docs/communities/moderation/_index.md

```diff
@@ -95,21 +93,280 @@ in the message bar at the bottom:
+- `@john:example.com` is the Matrix ID of the person we want to ban
+- `User keeps insulting people` is the reason why we are banning them. Their
+   client will be able to display that message.
+
+Mjolnir then adds the user we're banning to the ban list we specified. Mjolnir
+keeps watching one or several ban lists, and will ban all the users present on
+those lists from all the rooms it protects.
+
+Note that this command will likely fail if it's the first time you are trying to
+ban someone. Indeed, you need to create one or more ban lists before adding
+users or servers on them. See the section [Creating ban lists](#creating-ban-lists)
+below for more information on how to create them.
+
+### Removing a server definitely
+
+Matrix is a federated network. This means people can set up new servers
+specifically to harm some communities. This allows them to create a virtually
+infinite number of users, making it possible to raid other communities easily.
+
+Fortunately, all these malicious users have one thing in common: they come from
+the same homeserver. This means if you block the malicious homeserver entirely,
+all the malicious users are going to be banned at once.
+
+To do so, you can issue a variant of the command used to ban an individual user.
+The command was
+
+```
+!mjolnir ban <list shortcode> <user|room|server> <glob> [reason]
+```
+
+As you probably guessed, you are not going to ban a user but a server this once.
+For example, to ban the `maliciousdomain.tld` domain:
+
+```
+!mjolnir ban spam server maliciousdomain.tld
+```
+
+Mjolnir is going to add the domain to the `spam` ban list, and whenever someone
+from that homeserver tries to join your community they are going to be banned.
+
+### Creating ban lists
+
+Mjolnir allows you to create several ban lists that it can later watch. Those
+ban lists can also be watched by third parties if you make them public.
+
+The command to issue in mjolnir's control room to create a list is the
+following
+
+```
+!mjolnir list create <shortcode> <alias localpart>
+```
+
+- `shortcode` is a short name given to this list. It should be short and easy to
+  type, since you are likely to type it a lot when banning users.
+- `alias localpart` is the local part of the address mjolnir is going to create
+  for this list. This is useful if you ever want to [share your ban list with
+  other communities](#publishing-ban-lists).
+
+For example, assuming the server mjolnir is deployed against is `example.com`,
+the following command will create a ban list with the short code `spam` and the
+address `#my-community-spam-ban-list:example.com`
+
+```
+!mjolnir list create spam my-community-spam-ban-list
+```
+
+You only need to create a ban list once, and can then add as many users and
+servers as you want to that ban list. You can also create as many ban lists as
+you want.
+
+Communities often create two ban lists: one for spam, and one for code of
+conduct infringements.
+
```

Comment:
Maybe a mention how to set a list as a default would also be useful.

Reply:
Fixed in 3ecb5eca